yml/microsoft/built-in/wptsextensions.yml

```diff
@@ -0,0 +1,15 @@
+---
+Name: wptsextensions.dll
+Author: k4nfr3
+Created: 2022-08-15
+Vendor: Microsoft
+VulnerableExecutables:
+  - Path: 'Task Scheduler'
+    Type: Environment Variable

```

Comment:
I think this is actually Phantom DLL Hijacking, reading the blog more closely. It ends up checking %PATH% because the DLL does not exist at all